libs/langchain/langchain/vectorstores/qdrant.py

```diff
@@ -89,6 +89,7 @@ class Qdrant(VectorStore):
     def __init__(
         self,
         client: Any,
+        async_client: Optional[Any],
         collection_name: str,
         embeddings: Optional[Embeddings] = None,
```

Comment:
```suggestion
        client: Any,
        collection_name: str,
        async_client: Optional[Any] = None,
        embeddings: Optional[Embeddings] = None,
```

Not providing `=None` makes `async_client` mandatory, which I believe we don't want. 

Also try to see if you can use proper types (i.e. `async_client: Optional[AsyncQdrantClient]`)

Reply:
We can't do `Optional[AsyncClient]`, because the import of `qdrant_client` is done later within a try block. So we annotate it with a strict type down below.

https://github.com/langchain-ai/langchain/blob/cfffa02c3ca15673007b0cf1d51110cc44605007/libs/langchain/langchain/vectorstores/qdrant.py#L138

Also we'd want an `AsyncClient` passed everytime or `None` explicitly.